CONTRIBUTING.md

```diff
@@ -41,313 +27,228 @@ process documented here.
+Most [Kata Containers repositories](https://github.com/kata-containers)
+contain code written in the [Go language (golang)](https://golang.org/). Go 
+requires all code to be put inside the directory specified by the `$GOPATH` 
+variable. Follow this example to put the code in the standard location.
 
-    You do not (and cannot) make changes to this repository. PRs are a way of
-    allowing the project developers and administrators to review your work and
-    decide if they want to add your work to the upstream branch.
+```sh
+$ export GOPATH=${GOPATH:-$HOME/go}
+$ mkdir -p "$GOPATH"
+```
 
-    > **Note:** Don't worry - even if you make a mistake,
-    > you *cannot* do any damage to this repository.
+For further details on `golang`, refer to the 
+[requirements section of the Kata Developer Guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components).
 
-  - Your fork (`https://github.com/${your-github-username}/community`) on GitHub
+>*Note*: If you intend to make minor edits, it's acceptable
+> to simply fork and clone without adding the GOPATH variable.
 
-    This is where you send or upload (`git push`) your PR branches.
+##### Fork and clone
 
-  - Your *local copy* of the upstream repository
-    (https://github.com/kata-containers/community)
+In this example, we configure a Git environment to contribute to this very 
+`Community` repo. We create a sample branch, incorporate reviewer feedback, and rebase our commits.
 
-    This is where you actually make changes to your private copy of the
-    repository.
+1. Fork the [upstream repository](https://help.github.com/articles/cloning-a-repository):
 
-- [Setup a "git remote"](https://help.github.com/articles/configuring-a-remote-for-a-fork):
+1. [Clone your forked copy of the upstream repository](https://help.github.com/articles/cloning-a-repository):
 
-  "What happened to my fork?". The following demonstrates where your fork is
-  and how you use it.
+1. While on your *forked copy*, select the green button `Clone or download`
+   and copy the URL. 
 
-  Recall that your fork on the GitHub server is an *exact copy* of the
-  upstream repository that also lives on the GitHub server. That might not be
-  true moving forward because PRs are constantly merged into the upstream
-  repository. This means your fork will be out of date because it will not
-  contain the latest changes in the upstream repository. The same
-  problem also affects your local copy of the upstream repository.
+1. Run the commands below and **paste the copied URL** (previous step),
+   so your real GitHub user name replaces `your-github-username` below.
+  
+```sh
+$ dir="$GOPATH/src/github.com/kata-containers" 
+$ mkdir -p "$dir"
+$ cd "$dir"
+$ git clone https://github.com/{your-github-username}/community
+$ cd community
+```
+   
+**Note:** Cloning a forked repository automatically gives a remote `origin`.
```

Comment:
Strictly, this needs a leading `>` to make the note stand out. Same comment for some further notes below...

> **Note:** Cloning a ...

Reply:
Done.